src/main/node/config-manager.ts

```diff
@@ -80,3 +83,63 @@ function validateNonEmptyNorNull(property: string): void {
     throw new Error('This property cannot be null or empty');
   }
 }
+
+export function createAuthorFromCollaborators(
+  collaborators: Collaborator[] = []
+): string {
+  return collaborators.reduce(
+    (acc, { githubUsername }, index, { length }) =>
+      `${acc}${githubUsername}${index !== length - 1 ? ', ' : ''}`,
+    ''
+  );
+}
+
+export function modifyNodePackage(path: string, config: Configuration) {
+  const packagePath = `${path}/${NODE_PACKAGE_PATH}`;
+  validatePath(
+    !isDirectory(path) || !fs.lstatSync(packagePath).isFile(),
+    "Couldn't find the package.json file."
+  );
+
+  // eslint-disable-next-line import/no-dynamic-require
+  const pckgJson = require(packagePath);
```

Comment:
@BaranowskiBrt Seems like I did not add the comment to the last review before publishing it. But this should be

```ts
const pckgJson = JSON.parse(fs.readfileSync(packagePath).toString())
```
Otherwise the dynamic require will fail for files that are outside of the repository I think since they are not packaged before runtime. It's also good to keep this in line with how other files are accessed.

Sorry for the inconvenience. 🤦